### DISCUSSION TITLE
feature request: highlight only gitignore in `renderer.highlight_git`

### DISCUSSION THREAD

**Discussion by goolord** [nvim-tree/nvim-tree.lua#2028] [Ideas]
knowing that something is gitignores is useful for me, and highlighting them as a comment helps me filter them out as noise while still being able to interact with them in the filetree when i need to. but, it's not as useful or visually pleasing to see that some file is unstaged by making it red or something.

i'm currently doing this by just clearing all of the highlight groups used by `highlight_git` that don't deal with ignored files

**Comment by alex-courtis** [comment]
> i'm currently doing this by just clearing all of the highlight groups used by `highlight_git` that don't deal with ignored files

That is the answer: highlight groups are the mechanism for you to control exactly what is highlighted.

We won't be adding any configuration or options when the goal can be achieved via API or similar.
